projects/ngx-d3timeline/src/lib/scales/scale-utils.ts

```diff
@@ -37,11 +37,15 @@ export function rescaleTime(
 export function configureBandScale(
   activities: Activity[],
   view: View,
-  orientation: Orientation
+  orientation: Orientation,
+  lateralPadding: number
 ): BandScale {
   return scaleBand()
     .domain(getBandScaleDomain(activities))
-    .range(getRange(view, orientation));
+    .range(getRange(view, orientation))
+    .paddingInner(lateralPadding)
+    .paddingOuter(lateralPadding)
```

Comment:
not sure we need outer padding

Reply:
Its required to get even spacing on the edge resources. But see comment below maybe I'm misusing it.

With outer padding:
![image](https://user-images.githubusercontent.com/15385727/77466766-a4accf00-6e02-11ea-9d25-2dde1f749350.png)

Without outer padding:
![image](https://user-images.githubusercontent.com/15385727/77466817-c0b07080-6e02-11ea-936c-ded659d2d417.png)

Reply:
removed outer padding